libnmstate/netapplier.py

```diff
@@ -58,7 +59,7 @@ def _apply_ifaces_state(interfaces_desired_state, verify_change):
             _edit_interfaces(ifaces_desired_state, ifaces_current_state)
         if verify_change:
             # FIXME: Remove this sleep after adding wait for state mechanism.
-            time.sleep(2)
+            time.sleep(5)
```

Comment:
Is this intentional here? Maybe this should be its own commit with an explanation.

Reply:
No, a leftover. Thanks.